integration.py

```diff
@@ -65,6 +63,7 @@ def post(route, data, content_type):
 
             client = Client()
         else:
+            import app
```

Comment:
If the resource folder doesn't exist, then the import will fail at the top-level. There was a chicken-or-egg problem going on here, so I just moved it to the appropriate place in the Reporter class.